internal/resourcepolicies/resource_policies.go

```diff
@@ -29,6 +29,8 @@ const (
 	// currently only support configmap type of resource config
 	ConfigmapRefType string           = "configmap"
 	Skip             VolumeActionType = "skip"
+	FSBackup         VolumeActionType = "fs-backup"
+	Snapshot         VolumeActionType = "snapshot"
```

Comment:
Can you comment in code here the meaning of snapshot?

From the design following were also mentioned, it might be hard to distinguish between them all without explaining what snapshot above means as it relates to native, CSI, or data mover enablement.

> - `snapshot-native`: only use volume snapshotter (native cloud provider snapshots), do nothing if not present/not compatible
> - `snapshot-csi`: only use csi-plugin, don't use volume snapshotter(native cloud provider snapshots), don't use datamover even if snapshotMoveData is true
> - `snapshot-datamover`: only use csi with datamover, don't use volume snapshotter (native cloud provider snapshots), use datamover even if snapshotMoveData is false

Reply:
@kaovilai "snapshot" means that velero will either make a CSI snapshot or a native snapshot -- choosing between them will use the same logic Velero currently uses for non-fs-backup volumes when deciding whether to use CSI or native snapshots. Data mover enablement is a separate decision -- if a CSI snapshot is created and snapshotMoveData=true, then data movement will occur for this volume.

The three potential future values will override the default behavior described above. 
- `snapshot-native` will force using the native volume snapshotter, even if the volume would otherwise use CSI.
- `snapshot-csi` will force using CSI, even if the volume would otherwise use native snapshots (although I don't think there are currently any scenarios where CSI is enabled and available for this volume where the default behavior is native snapshots), and it will prevent data movement of the volume, even if `snapshotMoveData=true`
- `snaphot-datamover` will force using CSI (if available for the volume) and then use data movement, even if `snapshotMoveData=false`

Reply:
Right. I just kind of want that in the code comment.

Reply:
Done, added the relevant comments for action types